File: yeongseong/3week/두 수의 나눗셈.js
function solution(num1, num2) {
    // num1과 num2를 나눈 뒤 1000을 곱해준 값을 number에 저장
    const number = num1/num2*1000;
    // number의 정수 부분을 essence에 저장
    const essence = parseInt(number);
    return essence;
}

File: minwook/3주차/두_수의_나눗셈.java
class Solution {
    public int solution(int num1, int num2) {
        return (int)((float)num1 / (float)num2 * 1000); 
    }
} // float 형으로 적은 메모리로 num1, num2 형변환 나누고 1000을 곱한 뒤, 정수로 형변환